release-notes/logstash-output-opensearch-release-notes-2.0.1.md

```diff
@@ -0,0 +1,12 @@
+## Version 2.0.1 Release Notes
+Compatible with OpenSearch 1.3.4, 2.1.0
```

Comment:
Should we say `OpenSearch 1.3+, 2.0+` or something to that effect?

Reply:
`1.3+` makes sense but we're not sure if releases `2.1+` would all compatible?